ted/src/main/java/frc/robot/commands/AllignRelativeToTagCommand.java

```diff
@@ -0,0 +1,129 @@
+// ************************************************************
```

Comment:
Seems like this is not in use in the codebase in this PR.  When will it be used?  Should it be added then?

Reply:
We might use this for autonomous, allowing us to have more accurate autos because if we bump something our the robot would still know where it is because of vision

Reply:
this command doesn't yet work. Pull this file out of the PR and put it on a separate branch for future testing

Reply:
removed